chainer/functions/normalization/l2_normalization.py

```diff
@@ -108,4 +110,6 @@ def normalize(x, eps=1e-5, axis=1):
         as :math:`x`.
 
     """
+    if axis is None:
+        axis = tuple(range(1, x.ndim))
```

Comment:
Do you want to change the default?  I missed this when I commented before.

Reply:
I made this change to follow line 83 to 84.
I thought this change is not so drastic because this function assumed 2D inputs before #7062.

But there are tests for multi-dimensional cases, so now I have to cancel this change and delete line 83&84.
https://github.com/chainer/chainer/blob/76915601060c703d65e9daa1572016714c7b3c27/tests/chainer_tests/functions_tests/normalization_tests/test_l2_normalization.py#L43-L53